test/per_file_coverage.sh

```diff
@@ -21,7 +21,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/network/dns_resolver:90.7"  # A few lines of MacOS code not tested in linux scripts. Tested in MacOS scripts
 "source/common/protobuf:94.8"
 "source/common/quic:91.8"
-"source/common/router:96.5"
+"source/common/router:96.3"
```

Comment:
Is there no way to maintain coverage of this new code? Is it going down because of lines of code we need in order to satisfy the compiler but that won't actually get hit?

Reply:
unfortunately the only way to test PANIC is with death test, which doesn't help coverage.
there's a follow-up issue to move these panics over sensible defaults and actual error handling and that should come with unit tests.  As this shows, we have WAY too many PANICs in the data plane, due to this macro being misleading